tests/docker/globus.ubuntu20.Dockerfile

```diff
@@ -37,11 +19,38 @@ RUN apt-get update && apt-get install -y curl \
     sudo \
     libfuse2 \
     libcurl3-gnutls \
-    lsof
+    lsof \
+    wget \
+    gnupg2 \
+    lsb-release \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/*
+
+#### Get and install iRODS repo ####
+RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
+    echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list && \
+    apt-get update
```

Comment:
Move apt-get update to the beginning of the next RUN block. If this pattern is repeated anywhere else, make the same change. We don't want cached package lists remaining between image layers.

Reply:
I think I have made the relevant changes.  I need to learn about how image layers work...

Reply:
The short version is that every individual command in a Dockerilfe (`RUN` blocks, for example, being a single command) creates a new layer that is cached as an untagged docker image. If you don't clean up after yourself in every `RUN` block, it can inflate the size of the layers and increases the likelihood that problems can occur if `docker build` happens to use stale cached layers.